front/gatsby/src/pages/onboarding.tsx

```diff
@@ -2,16 +2,27 @@
 // This software may be modified and distributed under the terms
 // of the Apache-2.0 license. See the LICENSE file for details.
 
-import { AccountsList, Circle, Modal, Transition, polkadotOfficialTheme } from '@substrate/ui-components/src';
+import { AccountsList, Breadcrumbs, Circle, Modal, Transition, polkadotOfficialTheme } from '@substrate/ui-components/src';
 import { AccountsContext } from '@substrate/context/src';
-import React, { useContext } from 'react';
+import { RouteComponentProps } from '@reach/router';
+import React, { useContext, useState } from 'react';
 
-export const Onboarding = () => {
+const ONBOARDING_STEPS = [
+  'stash',
+  'controller'
+]
+
+interface Props extends RouteComponentProps {};
+
+export const Onboarding = (props: Props) => {
```

Comment:
Could you add function return types?

We can add a stricter eslint later, but let's give our future selves less work

Reply:
yes